processor/resourcedetectionprocessor/README.md

```diff
@@ -17,6 +17,22 @@ details of which are currently pending confirmation in the OpenTelemetry specifi
     * host.name
     * os.type
 
+* By default `host.name` is being set to FQND if possible, and a hostname provided by OS used as fallback.
```

Comment:
Please remove the list bullet since it's related to the current list section (not a new one) :)

Reply:
I have removed the list bullet.